packages/excalidraw/components/PublishLibrary.tsx

```diff
@@ -6,7 +6,7 @@ import { t } from "../i18n";
 import Trans from "./Trans";
 
 import { LibraryItems, LibraryItem, UIAppState } from "../types";
-import { exportToCanvas, exportToSvg } from "../../utils";
+import { exportToCanvas, exportToSvg } from "../../utils/export";
```

Comment:
Nah let the `utils/index` be there coz the final plan is to  have a excalidraw/utils where multiple utilities will be there  and we might still want to support `import {xyz} from "@excalidraw/utils"` instead of `import {xyz} from "@excalidraw/utils/abc"

Reply:
ok